cypress/integration/Balances.spec.js

```diff
@@ -14,6 +13,6 @@ context('Balances', () => {
     cy.get(tid("wallets-continue")).contains("Continue").click();
 
     cy.get(tid("set-trade-from", tid("token-amount-value"))).contains(/8,999.... ETH/) // avoid asserting exact value here since it may easily change in future
-    cy.get(tid("set-trade-to", tid("token-amount-value"))).contains("1,000 DAI")
+    cy.get(tid("set-trade-to", tid("token-amount-value"))).contains("170 DAI")
```

Comment:
Why did you change that to 170 ?

Also in those kind of tests probably we should get the balances and not use hardcoded values.

If we feed the account with other balance this test will fail.